lib/app/components/atoms/chat_bubble.dart

```diff
@@ -1,6 +1,8 @@
 import 'package:favoritism_communication/app/styles/styles.dart';
-import 'package:chat_bubbles/chat_bubbles.dart';
+import 'package:flutter_chat_bubble/bubble_type.dart';
+import 'package:flutter_chat_bubble/chat_bubble.dart' as cb;
 import 'package:flutter/material.dart';
+import 'package:flutter_chat_bubble/clippers/chat_bubble_clipper_4.dart';
 
 class ChatBubble extends StatelessWidget {
```

Comment:
```suggestion
class CustomChatBubble extends StatelessWidget {
```
すいません、レビュー遅くなりました。。
前のPRで指摘するべきだったと思うんですが、パッケージと同じクラス名にする必要ないかなと思ったので
`Custom`つけて区別しましょう。

Reply:
ご指摘の通り修正しました！